CHANGELOG.md

```diff
@@ -8,12 +8,13 @@
   - Reinstated `ics23` dependency ([#854])
 - [ibc-relayer]
   - Change the default for client creation to allow governance recovery in case of expiration or misbehaviour. ([#785])
-  
+
 ### BUG FIXES
 
 - [ibc-relayer]
   - Fix pagination in gRPC query for clients ([#811])
   - Fix relayer crash when hermes starts in the same time as packets are being sent ([#851])
+  - Fix missing port information in `hermes query channels` ([#840])
```

Comment:
@soareschen you also need to add a link to the issue below (before the `## v0.2.0` header)

Reply:
Done